### PR TITLE
Style: Add period to doc param tags

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Style fix in doc `param` tags:
- change `para` to `param`
- remove extra start and tail spaces
- add period
- uppercase first char
- remove extra space and period

Please review commit by commit.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/remoting/commands/RemoveJob.cs [hunk 1]
```diff
@@ -46,10 +46,10 @@ public class JobCmdletBase : PSRemotingCmdlet
         /// Find the jobs in repository which match matching the specified names
         /// </summary>
         /// <param name="writeobject">if true, method writes the object instead of returning it
-        /// in list (an empty list is returned). </param>
-        /// <param name="writeErrorOnNoMatch">write error if no match is found</param>
-        /// <param name="checkIfJobCanBeRemoved">check if this job can be removed</param>
-        /// <param name="recurse">recurse and check in child jobs</param>
+        /// in list (an empty list is returned). .</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/remoting/commands/RemoveJob.cs]: Double period.  Just remove whitespace.  Perhaps just globally search for ".  .</param>" and replace with ".</param>"

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/write.cs [hunk 2]
```diff
@@ -445,7 +445,7 @@ public WriteErrorException()
         /// <summary>
         /// Constructor for class WriteErrorException.
         /// </summary>
-        /// <param name="message"> </param>
+        /// <param name="message"> .</param>
```
[the remark this answers is not in the document]
Comment (iSazonov) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/write.cs]: Yes. I removed the space and period in last commit.

File: src/Microsoft.WSMan.Management/Interop.cs [hunk 3]
```diff
@@ -29,30 +29,30 @@ namespace Microsoft.WSMan.Management
     [TypeLibType((short)0)]
     public enum WSManEnumFlags
     {
-        /// <summary><param><c>WSManFlagNonXmlText</c> constant of <c>_WSManEnumFlags</c> enumeration.  </param><param>Constant value is 1.</param></summary>
+        /// <summary><param><c>WSManFlagNonXmlText</c> constant of <c>_WSManEnumFlags</c> enumeration.  .</param><param>Constant value is 1.</param></summary>
```
[the remark this answers is not in the document]
Comment (iSazonov) [reply src/Microsoft.WSMan.Management/Interop.cs]: I reverted the changes in the file.

File: src/System.Management.Automation/engine/remoting/commands/RemoveJob.cs (hunk 1 above)
Comment (iSazonov) [reply src/System.Management.Automation/engine/remoting/commands/RemoveJob.cs]: I reverted the changes in the file.

File: src/System.Management.Automation/help/MamlNode.cs [hunk 4]
```diff
@@ -22,13 +22,13 @@ namespace System.Management.Automation
     ///     1. para,
     ///         <param>
     ///             para text here
-    ///         </param>
+    ///         .</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/help/MamlNode.cs]: These examples should not have the period added

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/utils/ExecutionExceptions.cs [hunk 5]
```diff
@@ -95,7 +95,7 @@ public CmdletInvocationException()
         /// <summary>
         /// Instantiates a new instance of the CmdletInvocationException class
         /// </summary>
-        /// <param name="message"> </param>
+        /// <param name="message"> .</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/utils/ExecutionExceptions.cs]: Same as other one, does this actually reduce CodeFactor issues?  There's a bunch in this file.

File: src/System.Management.Automation/utils/MshArgumentException.cs [hunk 6]
```diff
@@ -34,7 +34,7 @@ public PSArgumentException()
         /// <summary>
         /// Initializes a new instance of the PSArgumentException class.
         /// </summary>
-        /// <param name="message"> </param>
+        /// <param name="message"> .</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/utils/MshArgumentException.cs]: Same as above, also a bunch in this file

File: src/System.Management.Automation/utils/MshNotSupportedException.cs [hunk 7]
```diff
@@ -78,8 +78,8 @@ public PSNotSupportedException(string message)
         /// <summary>
         /// Initializes a new instance of the PSNotSupportedException class.
         /// </summary>
-        /// <param name="message"> </param>
-        /// <param name="innerException"> </param>
+        /// <param name="message"> .</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/utils/MshNotSupportedException.cs]: Same

File: src/System.Management.Automation/utils/MshObjectDisposedException.cs [hunk 8]
```diff
@@ -25,7 +25,7 @@ public class PSObjectDisposedException
         /// <summary>
         /// Initializes a new instance of the PSObjectDisposedException class.
         /// </summary>
-        /// <param name="objectName"> </param>
+        /// <param name="objectName"> .</param>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/utils/MshObjectDisposedException.cs]: Same

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.Security/security/CertificateProvider.cs [hunk 9]
```diff
@@ -2360,7 +2360,7 @@ protected override object GetChildItemsDynamicParameters(string path, bool recur
         /// <summary>
         /// get the name of the specified certificate
         /// </summary>
-        /// <param name="cert"> .</param>
+        /// <param name="cert"></param>
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Security/security/CertificateProvider.cs]: We should probably change the other ones with a single space (or space with period) to just empty and (hopefully) get actual descriptions in over time

File: src/System.Management.Automation/utils/ExecutionExceptions.cs (hunk 5 above)
Comment (iSazonov) [reply src/System.Management.Automation/utils/ExecutionExceptions.cs]: @SteveL-MSFT There already is a commit to remove this.

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/utils/ExecutionExceptions.cs (hunk 5 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/utils/ExecutionExceptions.cs]: You can ignore if it's already been resolved, was reviewing each commit in order

**Comment by iSazonov** [comment]
Closed to fix many merge conflicts in #8542.

**Pull request closed without merging**